src/containers/Tenant/Schema/SchemaViewer/SchemaViewer.tsx

```diff
@@ -20,15 +28,82 @@ const SchemaViewerColumns = {
     key: 'Key',
     type: 'Type',
     notNull: 'NotNull',
+    familyName: 'Family',
+    preferredPoolKind: 'Media',
+    columnCodec: 'Compression',
 };
 
 interface SchemaViewerProps {
-    keyColumnIds?: number[];
-    columns?: TColumnDescription[];
+    className?: string;
     type?: EPathType;
+    path?: string;
+    withFamilies?: boolean;
 }
 
-export const SchemaViewer = ({keyColumnIds = [], columns = [], type}: SchemaViewerProps) => {
+function prepareOlapTableSchema(tableSchema: TColumnTableDescription = {}) {
+    const {Name, Schema} = tableSchema;
+
+    if (Schema) {
+        const {Columns, KeyColumnNames} = Schema;
+        const KeyColumnIds = KeyColumnNames?.map((name: string) => {
+            const column = Columns?.find((el) => el.Name === name);
+            return column?.Id;
+        }).filter((id): id is number => id !== undefined);
+
+        return {
+            Columns,
+            KeyColumnNames,
+            Name,
+            KeyColumnIds,
+        };
+    }
+
+    return {
+        Name,
+    };
+}
+
+function formatColumnCodec(codec?: EColumnCodec) {
+    if (!codec) return null;
+    if (codec === EColumnCodec.ColumnCodecPlain) return 'None';
+    return codec.replace('ColumnCodec', '').toLocaleLowerCase();
+}
+
+function comparePossiblyEmptyStrings(a?: string, b?: string): number {
+    if (a === undefined && b === undefined) return 0;
+    if (a === undefined) return 1;
+    if (b === undefined) return -1;
+
+    return a.localeCompare(b);
```

Comment:
Let's not use one line if statements, they differ from the code style throughout the project and less error prof when editing

Reply:
It should be handled by linter if restricted. There I do not see any way when it would be edited to multiline.